### PR TITLE
plugins:foreman_ansible - Release 0.3

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: plugins/ruby-foreman-ansible/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-ansible (0.3.0-1) stable; urgency=low
+
+  * 0.3.0 released
```
Comment (domcleal) [inline plugins/ruby-foreman-ansible/debian/changelog]: This should be 0.3 throughout.

File: plugins/ruby-foreman-ansible/debian/control [hunk 2]
```diff
@@ -2,12 +2,12 @@ Source: ruby-foreman-ansible
...
 Maintainer: Daniel Lobato Garcia <elobatocs@gmail.com>
-Build-Depends: debhelper, foreman (>= 1.6.0)
+Build-Depends: debhelper, foreman (>= 1.9.0), ruby-foreman-deface
 Standards-Version: 3.9.3
 Homepage: https://github.com/theforeman/foreman_ansible
 
 Package: ruby-foreman-ansible
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman (>= 1.6.0)
+Depends: ${misc:Depends}, bundler, foreman (>= 1.9.0), ruby-foreman-deface
```
Comment (domcleal) [inline plugins/ruby-foreman-ansible/debian/control]: `<< 2.0`?

**Comment by dLobatog** [comment]
Updated, reminder to self - don't release things without a .0 patch version

File: plugins/ruby-foreman-ansible/foreman_ansible.rb [hunk 3]
```diff
@@ -1 +1 @@
-gem 'foreman_ansible', '0.2.2'
+gem 'foreman_ansible', '0.3.0'
```
Comment (domcleal) [inline plugins/ruby-foreman-ansible/foreman_ansible.rb]: here too!

**Comment by dLobatog** [comment]
Updated,  didn't know the Gemfile using 0.3.0 instead of 0.3 would cause any problems?

**Comment by mmoll** [comment]
I didn't test, but guess it would lead to the plugin gem not being found for loading into foreman.

**Comment by mmoll** [comment]
cherry-picked

as | into
--- | ---
cb79925 | develop
ec020eb | 1.10
95a4714 | 1.9

gracias @dLobatog!

**Pull request closed without merging**